rabbit_mq/output.tf

```diff
@@ -11,6 +11,17 @@ output "broker_url" {
 }
 
 output "broker_user" {
-  value     = aws_mq_broker.mq.user
-  sensitive = true
+  value = aws_mq_broker.mq.user
 }
+
+output "broker_amqp_endpoint" {
+  value = aws_mq_broker.mq.instances.0.endpoints.0
+}
+
+output "broker_aws_secret_name" {
+  value = aws_secretsmanager_secret.broker_password.name
+}
+
+output "broker_console_endpoint" {
```

Comment:
This is already provided by `broker_url`

Reply:
removed `broker_url` in favor of this for descriptiveness